deeprank/models/variant.py

```diff
@@ -76,6 +82,10 @@ def insertion_code(self):
     def protein_accession(self):
         return self._protein_accession
 
+    @property
+    def enst_accession(self):
+        return self._enst_accession
+
```

Comment:
I see there is no hash function for enst_ac. Is this ok?

Reply:
It's a hard decision: the hash is used to identify the variant. Currently, the variant is identified by the pdb residue and the amino acid swap and not by optional uniprot/ENST accessions. We expect this not to matter though, because a variant that swaps 101M residue 10 by a tyrosine would always come from the same transcript and have the same uniprot entry, whether those are provided or not. I therefore chose variants to be defined solely on PDB data, but it could have been done otherwise.